docs/guidelines/FAQ/pre-commit-faq.md

```diff
@@ -25,7 +25,8 @@ Pre-commit can be installed in two ways, using Python pip or Homebrew.
 |---|---|
 | `pip install pre-commit` | `brew install pre-commit` |
 
-Once pre-commit is installed you need to set up the git hook scripts by running `pre-commit install`. Now pre-commit wil automatically run on git commit!
+Once pre-commit is installed you need to manually add it to PATH (user variables). 
+Set up the git hook scripts by running `pre-commit install`. Now pre-commit wil automatically run on git commit!
```

Comment:
I tried to find others with the same issue, but could not immediately find any documentation that updating the PATH is _generally_ needed.

Might this be a "local" issue, depending on _how_ you installed `pre-commit`? 🤔 
If so, I would probably changed the text to something like
````md
Set up the git hook scripts by running `pre-commit install`. Now pre-commit wil automatically run on git commit!

``` info
  If the `pre-commit` is not available after install, it might be that you need to manually add it to PATH (user variables). 
```
````

Reply:
Indeed might not be general. Since I use pip-install, I am unsure how brew might work.